__tests__/test.js

```diff
@@ -66,3 +69,27 @@ describe("test DB for requests", () => {
       });
   });
 });
+
+describe("GET /api/articles ", () => {
+  test("/api/articles returns all article in order of date", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((res) => {
+        expect(res.status).toBe(200);
+        expect(res.body.length).toBeGreaterThan(0);
+        expect(res.body).toBeSorted("comments.created_at");
+        expect(res.body).toBeSorted({ descending: true });
+        res.body.forEach((topic) => {
+          expect(topic).toHaveProperty("title");
+          expect(topic).toHaveProperty("article_id");
+          expect(topic).toHaveProperty("author");
+          expect(topic).not.toHaveProperty("body");
+          expect(topic).toHaveProperty("topic");
+          expect(topic).toHaveProperty("created_at");
+          expect(topic).toHaveProperty("votes");
+          expect(topic).toHaveProperty("article_img_url");
+          expect(topic).toHaveProperty("comment_count");
```

Comment:
Ideally we would have firmer checks here checking that the data type on each property is what we expect it to be. I would love you to look into refactoring this so we're asserting the property is there and the data type on the property is correct. 

Certain jest matchers make our life so much better like `toMatchObject` or `objectContaining`. Have a look on the internet at how these kinds of matchers are used to do a load of assertions on an object in a DRYer way